### PR TITLE
Feat/add oif summit 2023 proposal cloud repatriation

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Cloud Repatriation is a term that has been buzzing around the cloud industry since
+the paper by a16z and has led to various claims that the age of cloud has its rise.
+With efforts such as the Sovereign Cloud Stack Project more standardiziation of Cloud
+offerings by Cloud Service Providers is anticipated. Such efforts are a big chance
+for OpenStack since it allows that even in cases of Cloud Repatriation OpenStack will
+still be the chosen workhorse, just not in a public but in a private cloud environment.
+This talks wants to revisit the myths of cloud computing and what it really
+has to offer aside from the always-promised easy scaling.
+In the age of Infrastructure-as-Code standardized environments where the IaC
```
Comment (itrich) [inline OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: Hm, this is a really long sentence. I suggest to split it (for easier reading).

> In the age of **standardized** Infrastructure-as-Code environments, OpenStack can offer a lot of benefits – especially if shifted from public cloud to private environments. Backed by shared knowledge and thus fostering operational confidence, the underlying platform can be kept and well-defined standards will assure interoperability.

Or something like this.

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Authors: Felix Kronlage-Dammers
+
+Technical Level: Intermediate
+
+## Abstract (1000 chars)
+
+Cloud Repatriation is a term that has been buzzing around the cloud industry since
+the paper by a16z and has led to various claims that the age of cloud has its rise.
+With efforts such as the Sovereign Cloud Stack Project more standardiziation of Cloud
```
Comment (itrich) [inline OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: of cloud offerings

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+offerings by Cloud Service Providers is anticipated. Such efforts are a big chance
+for OpenStack since it allows that even in cases of Cloud Repatriation OpenStack will
+still be the chosen workhorse, just not in a public but in a private cloud environment.
```
Comment (itrich) [inline OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: will remain the chosen workhorse

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+of benefits.
+
+## Social Summary (100 chars)
+
+Cloud Repatriation as a chance for more OpenStack adoption - let's move.
+
+## What should attendees expect to learn? (1000 chars)
+
+Even though various well-known people blog and tweet about how they left
+the cloud it is not the end of cloud computing. Attendees will have
```
Comment (itrich) [inline OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: how they left the cloud **,** it is not the

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+the chance to see how complex is this topic really is and why standardization
```
Comment (itrich) [inline OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: how complex ~~is~~ this topic really is

**Review by itrich — CHANGES_REQUESTED** [review]

File: OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md (hunk 1 above)
Comment (fkr) [reply OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md]: oh. @itrich! very good suggestion. thanks!

**Review by itrich — APPROVED** [review]
LGTM!

**Pull request merged**